Product/Production/Services/AuditRepositoryCore/src/test/resources/hibernate/auditrepo.hibernate.cfg.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<!DOCTYPE hibernate-configuration PUBLIC "-//Hibernate/Hibernate Configuration DTD//EN" "http://hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<!DOCTYPE hibernate-mapping SYSTEM "classpath://org/hibernate/hibernate-configuration-3.0.dtd">
```

Comment:
Yes, I accidentally pasted the "-mapping" DOCTYPE over to the .cfg's in the test package.